packages/cli/package.json

```diff
@@ -54,7 +54,7 @@
     "rimraf": "3.0.2",
     "strip-ansi": "6.0.0",
     "ts-jest": "27.0.4",
-    "ts-node": "10.1.0",
+    "ts-node": "10.2.0",
```

Comment:
I don't understand why we need ts-node here. It is only used by the `@coveo/search-token-server` package.

Reply:
Indeed, don't hesitate to do a PR to remove dependencies that are not used 👍